### DISCUSSION TITLE
Does schedule affects Dashboard / MR actions?

### DISCUSSION THREAD

**Discussion by chkpnt** [renovatebot/renovate#15937] [Request Help] [answered]
I'd like to run Renovate for my project on a weekly basis, but I'm unsure how it affects actions triggered by the checkmarks in the dashboard and merge-/pull-requests. It would be nice if those actions are executed as soon as possible and do not wait for the next scheduled run. As I haven't tried it out yet, what is the behaviour regarding those actions when a schedule is configured?

**Comment by HonkingGoose** [comment, marked as answer]
Quote from our docs: [^docs]

> Examples of what having a Dependency Dashboard will allow you to do:
> 
> - View all PRs in one place, rather than having to filter PRs by author
> - Rebase/retry multiple PRs without having to open each individually
> - Override any rate limiting (e.g. concurrent PRs) or scheduling to force Renovate to create a PR that would otherwise be suppressed
> - Recreate an unmerged PR (e.g. for a major update that you postponed by closing the original PR)

You can use the dashboard to get updates ahead of your schedule. If you click on a update in the dashboard, you should get the update PR soon after. 😉 

[^docs]: [Renovate docs, `dependencyDashboard`](https://docs.renovatebot.com/configuration-options/#dependencydashboard)

**Reply by chkpnt** [reply]
Thanks! I filed a PR (https://github.com/renovatebot/renovate/pull/15941) regarding the docs.

And thanks to your answer I'm now aware that GitHub Markdown supports [footnotes](https://github.blog/changelog/2021-09-30-footnotes-now-supported-in-markdown-fields/)! 🎉

**Reply by HonkingGoose** [reply]
Thanks for opening a PR to improve the docs! ❤️ 

Yeah, footnotes are nice. 😄
